src/NzbDrone.Api/Config/HostConfigResource.cs

```diff
@@ -22,6 +22,8 @@ public class HostConfigResource : RestResource
         public string Branch { get; set; }
         public string ApiKey { get; set; }
         public string SslCertHash { get; set; }
+        public string SslCertPath { get; set; }
+        public string SslCertPassword { get; set; }
```

Comment:
Need to add these to new API resource as well. Radarr.Api.V2. This api is not used by UI